### PR TITLE
Adds cartridge pouch for blackpowder weapons

### PR DESCRIPTION
**Pull request by DangerNoodle**
Added a cartridge pouch for paper cartridges, in the vein of a giberne or gargousier. Capacity is 14, directly in between the rifle and shotgun bandolier capacities. General characteristics are designed with it being worn at the waist, like the ammo satchel (which technically holds a magazine).

The only oddity is adding blunderbuss rounds to its list of accepted cartridges. Despite the volume disparity, the blunderbuss is already unlikely to see much use by players, and I see no harm in allowing the cartridge pouch to hold them as well. It can always be removed if desired.

Additionally added it to museum firearm itemgroup, and to the reenactor professions.

And finally, an itemgroup addition that I forgot to give the double-barrel flintlock.

### CODE REVIEW

**Comment by DangerNoodle** [comment]
Additionally, an alternative. I could instead add the cap-and-ball rounds to pistol bandoliers, paper cartridges to rifle bandoliers, and blunderbuss rounds to shotgun bandoliers. The pistol bandolier (being explicitly cowboy-themed) is the only instance where this is appropriate.

Whichever method sounds more acceptable?

**Comment by BorkBorkGoesTheCode** [comment]
That could work, you would need to find a way to safely store the percussion caps.
https://en.wikipedia.org/wiki/Paper_cartridge

**Comment by DangerNoodle** [comment]
Whereas it would be assumed that a cartridge pouch is designed to hold any relevant parts. And dating further back, bandoliers back before paper cartridges mostly just stored vials of powder, if I recall correctly.

So a separate item for these ammo types might be more logical, gameplay-wise I do not suspect it matters much. At least I hope.

**Comment by mugling** [comment]
Largely dependent upon the outcome of your other PR - consider merging the two

**Comment by DangerNoodle** [comment]
The one I mentioned wherein I rebalance the cap-and-ball rounds? By now my assorted ideas might further make that pull request more extensive:

* Whether or not to make black gunpowder autolearned.
* Reduce ratio of black powder to normal gunpowder to 1.5.
* Add blackpowder to shotgun shells, .22 LR, shotshells.
* Add way to mold shotshells (possibly with plastic mod, hand press).

**Comment by DangerNoodle** [comment]
Granted, increasing the damage of cap-and-ball ammo to be less inferior (yet still lower) than their closest modern equivalents is a minor change, and I was thinking that rebalancing is better than lumping them together.

**Comment by mugling** [comment]
> The one I mentioned wherein I rebalance the cap-and-ball rounds?

Yes, this idea is dependent on that so should either be part of that PR or resubmitted after it - your choice either way

**Comment by DangerNoodle** [comment]
I can implement the rebalance in this pull request, rather than making a separate pull request and moving this content to it.

My main concern behind holding off on that pull request was the gunpowder and related changes, as needing to port over that recipe might conflict with https://github.com/CleverRaven/Cataclysm-DDA/pull/18549.

However, rebalancing is presently dependent on https://github.com/CleverRaven/Cataclysm-DDA/pull/18549, given my statement that I will balance them after

**Comment by DangerNoodle** [comment]
And it seems I need to fix a merge conflict in my aluminum pull request anyway.

**Comment by DangerNoodle** [comment]
Since I do not plan on removing or combining the revolver paper cartridges, this pull request should hopefully not be dependent on planned rebalancing. Which is good, as I can not start that until the pepperbox fix is merged.

File: data/json/itemgroups/guns.json [hunk 1]
```diff
@@ -220,7 +220,8 @@
       [ "rifle_22", 60 ],
       [ "rifle_3006", 30 ],
       [ "rifle_9mm", 60 ],
-      [ "surv_carbine_223", 120 ]
+      [ "surv_carbine_223", 120 ],
+      [ "carbine_flintlock_double", 30 ]
```
Comment (mugling) [inline data/json/itemgroups/guns.json]: alphabetical ordering (this avoids merge conflicts as otherwise everyone tries to append to the end)

**Pull request merged**